src/components/InputSection/InputSection.js

```diff
@@ -0,0 +1,27 @@
+export default class InputSection {
+  constructor(keypressEvent) {
```

Comment:
```suggestion
  constructor(onKeyDown) {
```
컴포넌트마다 이벤트 파라미터를 {이벤트명}Event로 주고 계신것 같은데, 함수라면 동사로 네이밍 하도록 통일하면 좋을것 같습니다!
처음에 keypressEvent 객체라고 생각했어욯ㅎㅎ

+) cos에서는 이벤트 리스너 함수명을 on{이벤트명}Handler로 통일하고 있습니다

Reply:
앗 혼란을 드렸군용 예시 감사합니닷!